panel-app/src/components/file-explorer/file-explorer.tsx

```diff
@@ -159,7 +159,7 @@ export class FileExplorer {
 						this.recentFiles = this.recentFiles.filter((item) => item !== filename);
 						this.fileList = this.fileList.filter((item) => item !== filename);
 
-						logEvent('deleted file');
+						logEvent('deleted file', getScraperConfigMetrics());
```

Comment:
Why are we logging metrics for a deleted file?

Reply:
Why not? we want data!! all the data. 😄 

For times when the file wasn't used in a while and then simply got deleted. I would like to see them. 
Just in case we need to see more... It's easier to ignore that set of data than to modify the events and wait for new analytics.